### PR TITLE
 fix to issue: vllm model caching #2 

### LINKED ISSUE #2: vllm model caching
Currently, vllm modelspec class definition does not cache the models when downloaded. We need to establish the caching variables correctly to solve this issue.

### PR DESCRIPTION
**Pull request by GhaziSyed**
Resolves #2

### CODE REVIEW

File: fmperf/ModelSpecs.py [hunk 1]
```diff
@@ -418,7 +425,7 @@ def __init__(
         tokenizer: Optional[str] = None,
         tokenizer_mode: str = "auto",
         trust_remote_code: bool = False,
-        download_dir: Optional[str] = None,
+        download_dir: Optional[str] = "/models",
```
Comment (tdoublep) [inline fmperf/ModelSpecs.py:428]: Can we revert this change?

File: fmperf/ModelSpecs.py [hunk 2]
```diff
@@ -517,6 +532,7 @@ def get_args(self):
         args += ["--dtype", self.dtype]
         args += ["--enforce-eager"]
 
+
```
Comment (tdoublep) [inline fmperf/ModelSpecs.py:535]: nit: unnecessary whitespace change

File: fmperf/WorkloadSpecs.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 class WorkloadSpec:
     def __init__(
         self,
-        sample_size: int = 10,
+        sample_size: int = 1,
```
Comment (tdoublep) [inline fmperf/WorkloadSpecs.py:7]: I think these changes belong to the other PR?

File: examples/workload_specifications.yml [hunk 4]
```diff
@@ -1,7 +1,4 @@
-min_input_tokens: 500
-max_input_tokens: 500
-min_output_tokens: 50
-max_output_tokens: 50
-frac_greedy: 1.0
-sample_size: 1
+input_tokens: 500
+output_tokens: 50
+greedy: True
```
Comment (tdoublep) [inline examples/workload_specifications.yml:3]: These changes belong in a different PR I think

**Review by tdoublep — COMMENTED** [review]
I added some minor changes. The main thing is to factor our the changes for the other issue into a different PR>

File: fmperf/ModelSpecs.py (hunk 1 above)
Comment (GhaziSyed) [reply fmperf/ModelSpecs.py:428]: Sure

File: fmperf/ModelSpecs.py (hunk 2 above)
Comment (GhaziSyed) [reply fmperf/ModelSpecs.py:535]: Ok

File: examples/workload_specifications.yml (hunk 4 above)
Comment (GhaziSyed) [reply examples/workload_specifications.yml:3]: i actually created 2 branches. let me see how i can fix it.

Comment (tdoublep) [reply examples/workload_specifications.yml:3]: ok - let me know if i can help

**Review by tdoublep — APPROVED** [review]
LGTM
